src/apps/miscapps/h3ToGeoBoundaryHier.c

```diff
@@ -17,7 +17,8 @@
  * @brief takes an H3 index and generates cell boundaries for all descendants
  * at a specified resolution.
  *
- *  usage: `h3ToGeoBoundaryHier H3Index [resolution outputMode]`
+ *  usage: `h3ToGeoBoundaryHier --parent parent [--resolution res] [--kml
```

Comment:
Looks like we're somewhat inconsistent about these `usage` comments. I think the best option would be something like
```
See `h3ToGeoBoundaryHier --help` for usage.
```
to avoid getting out of sync with the argument defs.